src/Interrupt_Sources.md

```diff
@@ -52,10 +52,8 @@ hidden by the text box.
 
 ## INT $50 — Timer interrupt
 
-Every time that the timer overflows (that is, when [TIMA](<#FF05 — TIMA: Timer counter>) exceeds $FF),
-an interrupt is requested by setting bit 2 in the IF register
-($FF0F). As soon as that interrupt is enabled, the CPU will execute it by
-calling the timer interrupt vector at $0050.
+The timer interrupt is requested every time that the timer overflows
```

Comment:
I think making the words "is requested" into a link to either `IF`'s documentation or a more general description of the IRQ process (is there one?) would be appropriate in fulfilling the role that the rightfully removed text was. What do you say?

Reply:
Agree!